### DISCUSSION TITLE
How to get active / highlight state for index page in sidebar

### DISCUSSION THREAD

**Discussion by OCram85** [vuejs/vitepress#3328] [Q&A]
Hey folks,
just started setting up vitepress with multiple sidebars and noticed the `index` page never gets marked / highlighted as an active page.

![image](https://github.com/vuejs/vitepress/assets/13993700/2238d728-4383-4d87-9807-8c26ade79afb)

It doesn't matter if use the url `<foobar>/pwsh/` or `<foobar>/pwsh/index`

- external `sidebar.mjs` definition:

```js
// ....
// ....
// ....

function getPwsh() {
  return [
    {
      text: 'Introduction',
      items: [
        { text: 'About', link: 'index' },
        {
          text: 'Style Guide',
          items: [
            { text: 'Rules', link: 'style-guide/' },
            { text: 'Formatting & Style', link: 'style-guide/format' },
            { text: 'Source Files', link: 'style-guide/files' },
          ]
        }
      ]
    }
  ]
}

// ....
// ....
// ....

export default {
  // ...
  Pwsh: getPwsh(),
 // ...
}
```

- vitepress `config.mjs`:

```js
import sidebar from './sidebars.mjs'
// ...
// ...
sidebar: {
    // ...
      '/pwsh/': { base: '/pwsh/', items: sidebar.Pwsh},
    },
    // ...
```
-- the repo  structure looks like this:

![image](https://github.com/vuejs/vitepress/assets/13993700/10bb06ec-dcde-45ff-b861-cf1ecd69e6f7)

Which link should be used for index (about) page in sidebars when der is already a `base` set? Already tested this:

- `{ text: 'About', link: 'index' },`
- `{ text: 'About', link: '.' },`
- `{ text: 'About', link: './' },` 
- ` { text: 'About', link: 'index' },`
- ` { text: 'About', link: './index' },`
- ` { text: 'About' },`

**Comment by Feelthis** [comment]
In my case problem solved by adding slash in the end of link:
{ text: 'About', link: '/about/' }

**Comment by brc-dd** [comment]
https://github.com/vuejs/vitepress/issues/3031#issuecomment-3579215943
